### DISCUSSION TITLE
kind of flickering PFD

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#5455] [General]
Hi,

does anybody else have a groovy looking PFD after the last MS uodate?! I see a lot of squares .... 

![grafik](https://user-images.githubusercontent.com/76531893/127471162-6b226906-b6ca-4c31-a922-5b2f9287a430.png)

The other displays are as well not very sharb, only when zooming in ... But I think this is a result of the update .... I am completly free of stuttering but the qualitiy becomes worser. Seems that they have reduced the LOD to get rid of the stuttering .... hmmm ...

Greets Klaus

**Comment by emsss78** [comment]
I confirm...this is the first thing I noticed after the update...when I zoom in the flickering is gone

**Comment by ExampleWasTaken** [comment]
This was accidentally implemented when pixelated screens where implemented. A reversion is already in the works.
